hack/build-integration-kube.sh

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+set -o errexit -o errtrace -o functrace -o nounset -o pipefail
+root="$(cd "$(dirname "${BASH_SOURCE[0]:-$PWD}")" 2>/dev/null 1>&2 && pwd)"
+readonly root
+# shellcheck source=/dev/null
+. "$root/scripts/lib.sh"
+
+GO_VERSION=1.22
+
+
+# Note that these are ONLY used to bootstrap the cluster, and are only installed if not already on the system
+KIND_VERSION=v0.23.0
+# Note that 1.7.6 does not work. No particular need to keep this in sync though.
+NERDCTL_VERSION=2.0.0-rc.0
```

Comment:
Shouldn't this be compiled from the source?

Reply:
This is only used by kind to bootstrap, so, it does not matter much.
I'll change it though - we have the binaries, I can just force the PATH for kind.